### ISSUE TITLE
How to look at bbox predictions after training?

### ISSUE THREAD

**Issue by alareza619** [pytorch/vision#1892]
Hi, I finetuned a pretrained Faster RCNN model.
I used the instance segmentation Mask RCNN pytorch tutorial as a guide.

I finished training and can't figure out how to look at bbox predictions.

For segmentation prediction, the guide used the following to display the mask
Image.fromarray(prediction[0]['masks'][0, 0].mul(255).byte().cpu().numpy())

I tried 
Image.fromarray(prediction[0]['boxes'].mul(255).byte().cpu().numpy()) 
But this doesn't work.

link to tutorial I followed : https://colab.research.google.com/github/pytorch/vision/blob/temp-tutorial/tutorials/torchvision_finetuning_instance_segmentation.ipynb#scrollTo=5v5S3bm07SO1

I used the following to train on a custom dataset. 
      

- def get_faster_rcnn_model(num_classes):
-     model = torchvision.models.detection.fasterrcnn_resnet50_fpn(pretrained=True)
-     num_classes = 2  # 1 class (person) + background
-     in_features = model.roi_heads.box_predictor.cls_score.in_features
-     model.roi_heads.box_predictor = FastRCNNPredictor(in_features, num_classes) 
- 
-     return model

**Comment by fmassa** [comment]
Hi,

I hope you have found how to visualize the boxes. The `prediction[0]['boxes']` corresponds to the `[x0, y0, x1, y1]` coordinates of the pixels in the image, so you can use `matplotlib.pyplot.Rectangle` to plot it
